application.go

```diff
@@ -304,10 +314,12 @@ func (client *Client) SetApplicationVersion(name string, version *ApplicationVer
 	client.log("SetApplicationVersion() setting the application: %s to version: %s", name, version)
 	uri := fmt.Sprintf("%s/%s", MARATHON_API_APPS, trimRootPath(name))
 	deploymentId := new(DeploymentID)
-	if err := client.apiPut(uri, version, deploymentId); err != nil {
+	err := client.apiPut(uri, version, deploymentId)
+	if err != nil {
```

Comment:
Agreed .... for single error returns it makes sense to keep the if err:= doSomething; err != nil { ... i'll revert that line